src/ds++/test/tstDracoMath.cc

```diff
@@ -17,7 +17,7 @@
 //---------------------------------------------------------------------------//
 
 void tstabs(rtt_dsxx::UnitTest &ut) {
-  using rtt_dsxx::abs;
+  using std::abs;
```

Comment:
I feel like the purpose of this is to test the overloads in ds++/DracoMath:

`// Specialization for standard types - There is no standard abs function for
// float -- one reason why we define this template!
template <> inline double abs(double a) { return std::fabs(a); }
template <> inline int abs(int a) { return std::abs(a); }
template <> inline long abs(long a) { return std::labs(a); }`

So this might be changing the intent / behavior of the test (even though the test still passes). It looks like it does decrease the coverage of the DracoMath header!

Reply:
I've removed `rtt_dsxx::abs` altogether.  I don't see why it is needed and `std::abs` seems to do everything we need.